packages/nimble-components/src/accordion/specs/README.md

```diff
@@ -0,0 +1,152 @@
+# Nimble Accordion
+
+## Overview
+
+The `nimble-accordion` is a vertical stack of interactive headings. The headings are controls that can be used to reveal or hide content in specific sections. The Nimble accordion is based upon [FAST's accordion component](https://github.com/microsoft/fast/tree/archives/fast-element-1/packages/web-components/fast-foundation/src/accordion).
+
+### Background
+
+[Nimble issue #533: nimble-accordion Component](https://github.com/ni/nimble/issues/533)
+
+[Visual Design](https://www.figma.com/file/PO9mFOu5BCl8aJvFchEeuN/Nimble_Components?type=design&node-id=1295-85131&mode=design&t=DxDRlqT7MeCPLOxi-0)
+
+Visual Design Comments:
+
+Brandon's Figma spec has designs for two accordion item heights, 24px and 32px. The 32px will be implemented as this is what we have previously focused on when choosing between the two sizes.
+
+Brandon's Figma spec also only shows designs for one accordion item at a time. How should accordion items in each appearance type be separated from each other (ex. a line between accordion items, transparent space between accordion items, etc.)? Because there's only one accordion item design at a time, it is also unclear if error-visible is a nimble-accordion-item specific state, or if it should affect all accordion items and be used in the nimble-accordion component.
+
+The Figma design does not have a design for the "error" focusVisible state. This might be confusing, as keyboard users who tab to the error accordion will suddenly see the red border and exclamation mark icon disappear in the focusVisible state.
+
+When should each `appearance` type be used? Will there be specific guidance on this, and if so, will it be similar to the guidance for other Nimble components that use the same `appearance` types (ex. the Nimble-Button)?
+
+## Design
+
+Accordion group, appearance ghost with an error-visible accordion item
+
+```html
+<nimble-accordion appearance="ghost">
+    <nimble-accordion-item>
+        <span slot="heading">Accordion one</span>
+        <nimble-checkbox></nimble-checkbox>
+        Accordion one content
+    </nimble-accordion-item>
+    <nimble-accordion-item expanded>
+        <span slot="heading">Accordion two</span>
+        <nimble-checkbox></nimble-checkbox>
+        Accordion two content
+    </nimble-accordion-item>
+    <nimble-accordion-item error-visible>
+        <span slot="heading">Accordion three</span>
+        <nimble-icon-exclamation-mark></nimble-icon-exclamation-mark>
```

Comment:
Including the exclamation mark here is a bit ambiguous.

Are we saying that clients have to provide their own exclamation mark to indicate the error-visible state? I don't think we'd want that; if it's part of the design then the accordion should provide it.

Or are we just showing that you can include an icon as content? If that's the goal then let's pick a different icon for the example.

Reply:
Yeah, that's true, I'd assume that we wouldn't want clients to change the error icon, and they shouldn't have to include it themselves. I'll update this in the next commit.